packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -813,18 +860,51 @@ contract Deploy is Deployer {
             rawGameType != GameTypes.PERMISSIONED_CANNON.raw(), "Deploy: Permissioned Game should be deployed by OPCM"
         );
 
-        _factory.setImplementation(
-            _params.gameType,
-            IDisputeGame(
-                DeployUtils.create2AndSave({
-                    _save: artifacts,
-                    _salt: _implSalt(),
-                    _name: "FaultDisputeGame",
-                    _nick: string.concat("FaultDisputeGame_", vm.toString(rawGameType)),
-                    _args: DeployUtils.encodeConstructor(abi.encodeCall(IFaultDisputeGame.__constructor__, (_params)))
-                })
-            )
-        );
+        if (rawGameType == GameTypes.SUPER_CANNON.raw()) {
+            _factory.setImplementation(
+                _params.gameType,
+                IDisputeGame(
+                    DeployUtils.create2AndSave({
+                        _save: artifacts,
+                        _salt: _implSalt(),
+                        _name: "SuperFaultDisputeGame",
+                        _nick: string.concat("SuperFaultDisputeGame_", vm.toString(rawGameType)),
+                        _args: DeployUtils.encodeConstructor(abi.encodeCall(IFaultDisputeGame.__constructor__, (_params)))
+                    })
+                )
+            );
+        } else if (rawGameType == GameTypes.PERMISSIONED_SUPER_CANNON.raw()) {
+            _factory.setImplementation(
+                _params.gameType,
+                IDisputeGame(
+                    DeployUtils.create2AndSave({
+                        _save: artifacts,
+                        _salt: _implSalt(),
+                        _name: "SuperPermissionedDisputeGame",
+                        _nick: string.concat("SuperFaultDisputeGame_", vm.toString(rawGameType)),
+                        _args: DeployUtils.encodeConstructor(
+                            abi.encodeCall(
+                                IPermissionedDisputeGame.__constructor__,
+                                (_params, cfg.l2OutputOracleProposer(), cfg.l2OutputOracleChallenger())
+                            )
+                        )
+                    })
+                )
+            );
```

Comment:
I'm not sure we should be deploying `PERMISSIONED_SUPER_CANNON` here. At least based on this check for the pdg - https://github.com/ethereum-optimism/optimism/blob/ab0f1c3a820287c364f6456529c4990e81bb814b/packages/contracts-bedrock/scripts/deploy/Deploy.s.sol#L860 

I'd imagine that check also applies to super permissioned games. If you could confirm this @maurelian.